compiler/damlc/daml-package-config/src/DA/Daml/Package/Config.hs

```diff
@@ -66,6 +68,25 @@ parseProjectConfig project = do
     pSdkVersion <- queryProjectConfigRequired ["sdk-version"] project
     Right PackageConfigFields {..}
 
+checkPkgConfig :: PackageConfigFields -> [String]
+checkPkgConfig PackageConfigFields {pName, pVersion} =
+  [ "WARNING: Package names should have the format ^[a-zA-Z]([a-zA-Z0-9\\-])*[a-zA-Z0-9]$. \
```

Comment:
I think we misunderstood each other. I didn't (only) mean to break the long string across multiple lines in the code itself but to break the actual output to the terminal into multiple lines (that's where the 80 came from).

Reply:
oh! ok, I'll do that too.